### ISSUE TITLE
Add GhostNet-600 to model zoo

### ISSUE THREAD

**Issue by fmahoudeau** [PistonY/ModelZoo.pytorch#3]
Hi there, while browsing the official [GhostNet](https://gitee.com/mindspore/mindspore/tree/master/model_zoo/research/cv/ghostnet) repository on Gitee.com, I found a potentially very interesting version of GhostNet termed **GhostNet-600** which claims **80.2 top1 accuracy on ImageNet** for only 0.6GFLOPS and 11.9M parameters. It is not mentioned in the paper but the source code is available on [Gitee](https://gitee.com/mindspore/mindspore/blob/master/model_zoo/research/cv/ghostnet/src/ghostnet600.py).
Could you reproduce this performance and provide latency information ?

**Comment by DevinXYang** [comment]
Sounds interesting,I'll try it later.

**Comment by DevinXYang** [comment]
Hi @fmahoudeau , I only get 77% top1 on ImageNet same train setting with GhostNet1.3x.

**Comment by fmahoudeau** [comment]
Well thanks for giving it a try ! We should ask for clarification from the authors...

**Comment by DevinXYang** [comment]
I think they may use more tricks while training, still hard to believe getting 80% top1 in such low MAdds.

**Comment by fmahoudeau** [comment]
Yes that's hard to believe, and if it's true then they might not want to open source that part of their research.
Nevertheless, can you please upload the weights ? 77% is still useful because that's 1.5 points better than RegNetY-600MF and GhostNets have faster inference speed on CPU.

**Comment by DevinXYang** [comment]
I'm sorry for some reason I can't get the weights.
You could train it by yourself with same setting as GhostNet1.3x if you need it.
Notice that GhostNet using `no_decay_bias` and `LabelSmoothing` by default but `RegNet` not, they training epochs are also different.
